src/tests/data_tests/test_1_annotation.py

```diff
@@ -5,7 +5,7 @@
 
 
 class TestAnnotationGET(BiothingsDataTest):
-    host = "mygene.info"
+    host = "localhost:8000"
```

Comment:
suggest to change it back to `mygene.info` and use:

```
TEST_HOST=localhost:8000 pytest tests
```
to set local dev server when needed.